app/javascript/packs/src/settings/components/account/InputEnabled.jsx

```diff
@@ -0,0 +1,165 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { FormattedMessage } from "react-intl";
+import { FormGroup, FormControl, ControlLabel, Button } from "react-bootstrap";
+
+import {
+  BORDER_LIGHT_COLOR,
+  PRIMARY_GREEN_COLOR,
+  PRIMARY_HOVER_COLOR
+} from "../../../../app/constants/colors";
+
+const StyledInputEnabled = styled.div`
+  border: 1px solid ${BORDER_LIGHT_COLOR};
+  padding: 19px;
+  margin: 20px 0;
+
+  input {
+    margin-bottom: 15px;
+  }
+
+  .btn-primary {
+    background-color: ${PRIMARY_GREEN_COLOR};
+    border-color: ${PRIMARY_HOVER_COLOR};
+    margin-right: 7px;
+    &:hover {
+      background-color: ${PRIMARY_HOVER_COLOR};
+    }
+  }
+`;
+
+const ErrorMsg = styled.div`color: red;`;
+
+class InputEnabled extends Component {
+  constructor(props) {
+    super(props);
+
+    if (props.inputType === "password") {
+      this.state = {
+        value: "",
+        value2: ""
+      };
+    } else {
+      this.state = {
+        value: this.props.inputValue
+      };
+    }
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleChange2 = this.handleChange2.bind(this);
+    this.handleUpdate = this.handleUpdate.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ value: event.target.value });
+  }
+
+  handleChange2(event) {
```

Comment:
You don't need to eval here :smile:  you can put additional attribute in the input field and the do
event.target.attribute name : event.target.value